formatTest/unit_tests/expected_output/syntax.re

```diff
@@ -1129,3 +1129,25 @@ foo(
   ~y=[@foo] -0.1n,
   ~z=[@foo] -0.1p
 );
+
+/* Smooth formatting of functions with callbacks as arguments */
+test("my test", () =>
+  doStuff());
+
+test("my test", () => {
+  let x = a + b;
+  let y = z + c;
+  x + y;
+});
+
+describe("App", () =>
+  test("math", () =>
+    Expect.expect(1 + 2) |> toBe(3)));
+
+List.map((aankomst) =>
+  renderUitslagenAankomst(
+    ~aankomst,
+    ~leden,
+    ~updateAankomst,
+    ()
+  ), Aankomsten.toList(aankomsten));
```

Comment:
I like how Prettier formats callbacks. Here's an example:

```javascript
List.map(foo, bar, baz, (abc, z) =>
  MyModuleBlah.toList(argument)
);

List.map(
  (abc, z) => MyModuleBlah.toList(argument),
  foo,
  bar,
  baz
);
```

I notice it does two things that are worth emulating:
1. It only "docks" the callback if it is the last arg.
2. The final closing paren is aligned with the opening line (in refmt implementation that would be parens formatted with `~inline:(true, false)` I believe.